python_scripts/linear_models.py

```diff
@@ -541,8 +547,9 @@ def f(x):
 # `StandardScaler` transforms the data such that each feature will have a mean
 # of zero and a standard deviation of 1.
 #
-# These scikit-learn estimators are known as transformers: they compute some
-# statistics and store them when calling `fit`. Using these statistics, they
+# These scikit-learn estimator is known as a transformer: it compute some
+# statistics (i.e the mean and the standard deviation) and store them in its attributes
```

Comment:
```suggestion
# This scikit-learn estimator is known as a transformer: it computes some
# statistics (i.e the mean and the standard deviation) and stores them as attributes
```

Reply:
https://github.com/INRIA/scikit-learn-mooc/pull/37#discussion_r469414147
I would suggest actually showing in the code, the attributes of the transformer after fitting, and showing that it does store mean and sd

Reply:
added in the text indeed

Reply:
I would suggest actually showing it in code e.g.,

```
transformer.fit(X, y)
transformer.mean
transformer.sd
```
I just think that showing in the code can help some people grasp how it works more.

Reply:
added in https://github.com/lucyleeow/scikit-learn-mooc/pull/2/commits/cbf010f288f535969ea4465fe18cc4c487f3fea7